src/components/accordion/__tests__/CdrAccordion.spec.js

```diff
@@ -59,7 +59,7 @@ describe('CdrAccordion', () => {
     })
     describe('when the accordion is closed', () => {
       it('sets maxHeight to be 0', () => {
-        expect(wrapper.vm.maxHeight).toBe(0);
+        expect(wrapper.vm.maxHeight).toBe('0');
```

Comment:
This had to be updated to the String "0". Something about it being the number 0 was making this not compute in the context of the SSR bundle. Internally its probably doing some kind of `if` logic on the value and 0 is computing as `false`.

Additionally, we might want to normalize this to always append the unit measurement to the string e.g. `0px` to be consistent